_store/hmr-it-ios-offline-tracker.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Kimai2 iOS offline tracker
+name: Kimai2 iOS app for offline tracking
+intro: "Kimai2 iOS app for offline tracking"
+developer: hmr-it
+date: "2019-08-20 08:10:00"
+icon: fab fa-app-store-ios
+price: "1,92€"
+download: https://apps.apple.com/de/app/kimai-2-offline-zeiterfassung/id1459757165
+tags:
+  - app
+---
+
+Synchronize customers, projects and activities on your smartphone and 
+track times directly on your smartphone - even without an Internet connection. 
+Once you have internet again, you can upload your times to your Kimai 2 server.
+
+## Requires
+
+Kimai 2, V1.1+
```

Comment:
App Store says "minimum 0.9"

Reply:
Will be updated in app store with next app update.